cl/scrapers/tasks.py

```diff
@@ -484,132 +480,37 @@ def extract_by_ocr(path: str) -> (bool, str):
-    # and use the FLP logo for the Publisher Logo. If it lacks a seal, use the
-    # Publisher logo for both the front cover and the Publisher logo.
-    try:
-        has_seal = seals_data[court.pk]["has_seal"]
-    except AttributeError:
-        # Unknown court in Seal Rookery.
-        has_seal = False
-    except KeyError:
-        # Unknown court altogether (perhaps a test?)
-        has_seal = False
-
-    flp_image_frames = [
-        3,  # "Front Cover". Complete list at eyed3/id3/frames.py
-        14,  # "Publisher logo".
-    ]
-    if has_seal:
-        with open(
-            os.path.join(seals_root, "512", "%s.png" % court.pk), "rb"
-        ) as f:
-            audio_file.tag.images.set(
-                3, f.read(), "image/png", "Seal for %s" % court.short_name
-            )
-        flp_image_frames.remove(3)
-
-    for frame in flp_image_frames:
-        with open(
-            os.path.join(
-                settings.INSTALL_ROOT,
-                "cl",
-                "audio",
-                "static",
-                "png",
-                "producer-300x300.png",
-            ),
-            "rb",
-        ) as f:
-            audio_file.tag.images.set(
-                frame,
-                f.read(),
-                "image/png",
-                "Created for the public domain by Free Law Project",
-            )
-
-    audio_file.tag.save()
-
-
 @app.task
 def process_audio_file(pk):
     """Given the key to an audio file, extract its content and add the related
     meta data to the database.
     """
     af = Audio.objects.get(pk=pk)
-    tmp_path = os.path.join("/tmp", "audio_" + uuid.uuid4().hex + ".mp3")
-    av_path = get_audio_binary()
-    av_command = [
-        av_path,
-        "-i",
-        af.local_path_original_file.path,
-        "-ar",
-        "22050",  # sample rate (audio samples/s) of 22050Hz
-        "-ab",
-        "48k",  # constant bit rate (sample resolution) of 48kbps
-        tmp_path,
-    ]
     try:
-        _ = subprocess.check_output(av_command, stderr=subprocess.STDOUT)
-    except subprocess.CalledProcessError as e:
-        print(
-            "%s failed command: %s\nerror code: %s\noutput: %s\n%s"
-            % (
-                av_path,
-                av_command,
-                e.returncode,
-                e.output,
-                traceback.format_exc(),
-            )
-        )
-        raise
-
-    set_mp3_meta_data(af, tmp_path)
-    with open(tmp_path, "rb") as mp3:
-        cf = ContentFile(mp3.read())
+        response = convert_and_clean_audio(af).json()
+        cf = ContentFile(response["content"])
         file_name = trunc(best_case_name(af).lower(), 72) + "_cl.mp3"
         af.file_with_date = af.docket.date_argued
         af.local_path_mp3.save(file_name, cf, save=False)
+        af.duration = response["duration"]
 
-    af.duration = eyed3.load(tmp_path).info.time_secs
+    except requests.Timeout:
```

Comment:
Take a look at the new code that you're working off of. I removed all of this error handling junk yesterday (and talked about it on Slack). Can you adapt your code to remove it too? Having these errors show up in Sentry should be better than having them get mysteriously logged to the DB as they were in the past.

Reply:
Adapt I will.

Reply:
This is updated to match the very simple sentry error handling in cl-scrape-opinions